src/handler/http/request/folders.rs

```diff
@@ -56,7 +56,7 @@ pub async fn create_folder(
 /// UpdateFolder
 #[utoipa::path(
     context_path = "/api",
-    tag = "Dashboards",
+    tag = "Folders",
```

Comment:
Going forward folders will be treated as independent of dashboards, so I think it makes sense for these routes to get their own tag in the docs.